core/profiles/standard/standard.install

```diff
@@ -157,6 +157,28 @@ function standard_install() {
       ),
       'is_new' => TRUE,
     ),
+    array(
+      'type' => 'card',
+      'name' => st('Card'),
+      'description' => st('Add a piece of content to be used for blocks and views. A card does not have a public path (hide path display). Example use-case: Place card content as "Existing content" block in a Layout.'),
```

Comment:
This is too much text for a description, and includes the "Add" verb which could be incorrect in many instances. Recommendation:
`Add a card for use on an existing page. Display as a block, or in a view.`

For reference, here it is along side the other two descriptions:
* `Add a page with static content, like the 'About' page.`
* `Add a blog post, news article, or press release.`
* `Add a card for use on an existing page. Display as a block, or in a view.`

Reply:
I added `Display as a block, or in a view.` to meet the same needs as the original text, but now that I see it, I think we could exclude that too if people prefer.